src/oled_display_node.cpp

```diff
@@ -55,9 +55,10 @@ int main(int argc, char** argv) {
 
 // Default lines to output text to from this module
 #define  DISP_LINE_HOSTNAME    0
-#define  DISP_LINE_IP_ADDR     1
-#define  DISP_LINE_BATT_VOLTS  3
-#define  DISP_LINE_MOTOR_POWER 5 
+#define  DISP_LINE_IP_WLAN     1
+#define  DISP_LINE_IP_ETH     3
+#define  DISP_LINE_BATT_VOLTS  5
+#define  DISP_LINE_MOTOR_POWER 7 
```

Comment:
I am glad you did not remove the forcing of one display type, that would have backed out an important change.

Reply:
Yeah I didn't really go into that, if there are any changes to it it's because of the indentation fix I ran.